.travis.yml

```diff
@@ -4,7 +4,7 @@ sudo: required
 
 env:
   - ci_docker=debian:buster-slim ci_distro=debian ci_suite=stretch
-  - ci_docker=debian:buster-slim ci_distro=debian ci_suite=stretch ci_configopts="--with-curl"
+  - ci_docker=debian:buster-slim ci_distro=debian ci_suite=stretch ci_configopts="--with-curl --with-libsodium" ci_pkgs="libsodium-dev"
```

Comment:
May I ask for additional test in different configuration with disabled GPG (`--without-gpgme`)?
So we need to test 3 build environments:
- GPG-only
- GPG + libsodium
- libsodium-only

This is that I do for checking local changes -- I discover some issues only in one of those configuration.

Reply:
Well, our existing CI is gpg-only for the most part because we're not pulling in libsodium-dev.  But yep, changed this.